internal/action/show.go

```diff
@@ -182,8 +183,8 @@ func (s *Action) showHandleOutput(ctx context.Context, name string, sec gopass.S
 	}
 
 	if pw == "" && body == "" {
-		if ctxutil.IsShowSafeContent(ctx) && !ctxutil.IsForce(ctx) {
-			out.Warning(ctx, "safecontent=true. Use -f to display password, if any")
+		if config.Bool(ctx, "core.showsafecontent") && !ctxutil.IsForce(ctx) {
+			out.Warning(ctx, "core.showsafecontent=true. Use -f to display password, if any")
```

Comment:
```suggestion
		if config.Bool(ctx, "core.safecontent") && !ctxutil.IsForce(ctx) {
			out.Warning(ctx, "core.safecontent=true. Use -f to display password, if any")
```

Seems the name didn't match that of the config code?

Reply:
We should be using `core.showsafecontent` everywhere.

Reply:
Actually it seems the name change is on purpose, but the migration path doesn't migrate correctly?